navigation.c

```diff
@@ -67,7 +66,7 @@ update_camera(Camera *cam, Config *config, Mouse *mouse, Vec2f window_size)
         cam->delta_scale -= cam->delta_scale * cam->dt * config->scale_friction;
     }
 
-    if (!mouse->dragging && (LEN(cam->velocity) > VELOCITY_THRESHOLD)) {
+    if (!mouse->dragging && LEN(cam->velocity) > VELOCITY_THRESHOLD) {
```

Comment:
Yeah I originally had these because my LEN macro was bad but I since fixed it and forgot to update